### PR TITLE
- Add My Education Benefits form to saved forms on My VA profile

### PR DESCRIPTION
**Pull request by cesaresantiago**
- Add entries to helpers.jsx so My Education Benefits form shows up on saved forms on profile as documented in https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/forms/save-in-progress

- Note: The form name was set to 22-1990-MEB, but this will change when policy and procedures gives us our actual new form name. This name is provisional to move ahead and test.

## Original issue(s)
department-of-veterans-affairs/va.gov-team#0000

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by micahchiang — APPROVED** [review]

File: src/platform/forms/constants.js [hunk 1]
```diff
@@ -30,6 +30,7 @@ export const VA_FORM_IDS = Object.freeze({
   FORM_COVID_VACCINATION_EXPANSION: 'COVID-VACCINATION-EXPANSION',
   FORM_26_1880: '26-1880',
   FORM_21_22A: '21-22a',
+  FORM_22_1990_MEB: '22-1990-MEB', // there will be a new formal form name that will be provided for my educations benefit. Setting to 22-1990-MEB this while the procedures team provides it.
```
Comment (erikphansen) [inline src/platform/forms/constants.js:33]: can you move this up with the other 1990s a few lines above?

File: src/applications/personalization/dashboard/helpers.jsx [hunk 2]
```diff
@@ -48,6 +49,7 @@ export const formBenefits = {
   [VA_FORM_IDS.FORM_20_0996]: 'Higher-level review',
   [VA_FORM_IDS.FORM_VA_2346A]: 'hearing aid batteries and accessories',
   [VA_FORM_IDS.FORM_5655]: 'financial status report',
+  [VA_FORM_IDS.FORM_22_1990_MEB]: 'education benefits',
```
Comment (erikphansen) [inline src/applications/personalization/dashboard/helpers.jsx:52]: please move this up with the other 1990 forms

Comment (erikphansen) [inline src/applications/personalization/dashboard/helpers.jsx:52]: please move this up with the other 1990 forms

**Review by erikphansen — CHANGES_REQUESTED** [review]
Please move the new 1990 entries so that they live alongside the other 1990 entries. Thanks!

File: src/platform/forms/constants.js (hunk 1 above)
Comment (cesaresantiago) [reply src/platform/forms/constants.js]: Done. Thanks.

File: src/applications/personalization/dashboard/helpers.jsx (hunk 2 above)
Comment (cesaresantiago) [reply src/applications/personalization/dashboard/helpers.jsx:52]: Done, thx.

Comment (cesaresantiago) [reply src/applications/personalization/dashboard/helpers.jsx:52]: Done, thx.

**Review by erikphansen — APPROVED** [review]

**Review by bkjohnson — APPROVED** [review]

**Pull request merged**